EntityFrameworkCore.Manipulation.Extensions/Internal/Extensions/SqlCommandBuilderExtensions.cs

```diff
@@ -36,12 +36,11 @@ internal static class SqlCommandBuilderExtensions
         }
```

Comment:
Are you able to test these before deployment? My concern is that there could be issues again that we don't foresee here

Reply:
We have the benchmarks in PPE and that's what we'll have to look out for

Reply:
...and this is reverting to the old way of doing it, so there should be no change between the old version and this one. But we'll see, it's hard to test at scale before.

Reply:
OK, but that is an expensive way of testing it. I was hoping there was a better option here...